### PR TITLE
deploy: srsran gnb ue

### PR DESCRIPTION
**Pull request by PapaySail**
UE connected to gNB srsRAN Project using ZeroMQ -- for Open5GS core.

### CODE REVIEW

**Review by mbaran0v — COMMENTED** [review]
please improve structure, i think this variant more simply and clearly
docs/deployments/deployment_name/
- charts/srsran-gnb/Chart.yaml
- charts/srsran-gnb/values.yaml
- charts/srsran-gnb/templates/
- Makefile
- values/srsran-gnb.yaml (here values for our example installation)
- README.md

**Comment by PapaySail** [comment]
> please improve structure, i think this variant more simply and clearly docs/deployments/deployment_name/

Done. @mbaran0v check it, please.

File: docs/deployments/srsran-gnb/Makefile [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+# eupf oci://registry-1.docker.io/edgecom/eupf
+upf:
+	helm $(HELM_ACTION) \
+		eupf ../../../.deploy/helm/eupf \
+		--version 0.4.0 \
+		--values $(HELM_VALUES_DIR)/eupf.yaml \
+		-n $(NAMESPACE) \
+		$(HELM_ARGS)
+
+srs:
```
Comment (mbaran0v) [inline docs/deployments/srsran-gnb/Makefile:41]: use published chart

File: docs/deployments/srsran-gnb/Makefile [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+calico:
+	kubectl apply -f $(HELM_VALUES_DIR)/../manifests/calico-pools.yaml
+	kubectl apply -f $(HELM_VALUES_DIR)/../manifests/calico-bgp.yaml
+	kubectl patch felixconfiguration default --type='merge' -p '{"spec":{"removeExternalRoutes":false}}'
+
+# eupf oci://registry-1.docker.io/edgecom/eupf
+upf:
+	helm $(HELM_ACTION) \
+		eupf ../../../.deploy/helm/eupf \
```
Comment (mbaran0v) [inline docs/deployments/srsran-gnb/Makefile:35]: use published chart

**Review by mbaran0v — APPROVED** [review]

**Pull request merged**